api/data_explorer/controllers/facets_controller.py

```diff
@@ -79,27 +79,27 @@ def facets_get(filter=None, extraFacets=None):  # noqa: E501
     :type extraFacets: List[str]
     :rtype: FacetsResponse
     """
-    extra_es_facets, extra_ui_facets = _process_extra_facets(extraFacets)
-    combined_es_facets = OrderedDict(extra_es_facets.items() + current_app.
-                                     config['ELASTICSEARCH_FACETS'].items())
+    extra_ui_facets = _process_extra_facets(extraFacets)
     combined_ui_facets = OrderedDict(extra_ui_facets.items() +
                                      current_app.config['UI_FACETS'].items())
     search = DatasetFacetedSearch(
-        elasticsearch_util.get_facet_value_dict(filter, combined_es_facets),
-        combined_es_facets)
+        elasticsearch_util.get_facet_value_dict(filter, combined_ui_facets),
+        combined_ui_facets)
     # Uncomment to print Elasticsearch request python object
     # current_app.logger.info(
     #     'Elasticsearch request: %s' % pprint.pformat(search.build_search().to_dict()))
     es_response = search.execute()
     es_response_facets = es_response.facets.to_dict()
     # Uncomment to print Elasticsearch response python object
-    # current_app.logger.info('Elasticsearch response: %s' % pprint.pformat(es_response_facets))
+    #current_app.logger.info(
+    #    'Elasticsearch response: %s' % pprint.pformat(es_response_facets))
     facets = []
-    for name, field in combined_ui_facets.iteritems():
+    for elasticsearch_name, field in combined_ui_facets.iteritems():
```

Comment:
nit: `facet_info` over `field`? And same comment about `facet_name` over `elasticsearch_name` to be more general.